expirationd.lua

```diff
@@ -310,48 +306,129 @@ end
---                                supported index types are TREE and HASH, using other types will result in an error.
---     * iterator_type         -- Type of the iterator to use, as string or box.index constant,
---                                for example, 'EQ' or box.index.EQ; default is box.index.ALL;
---                                see https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_index/pairs/.
---     * start_key             -- Start iterating from the tuple with this index value;
---                                or when iterator is 'EQ', iterate over tuples with this index value;
---                                must be a value of the same data type as the index field or fields,
---                                or a function which returns such value;
---                                if omitted or nil, all tuples will be checked.
---     * tuples_per_iteration  -- Number of tuples to check in one batch (iteration); default is 1024.
---     * atomic_iteration      -- Boolean, false (default) to process each tuple as a single transaction;
---                                true to process tuples from each batch in a single transaction.
---     * process_while         -- Function to call before checking each tuple;
---                                if it returns false, the task will stop until next full scan.
---     * iterate_with          -- Function which returns an iterator object which provides tuples to check,
---                                considering the start_key, process_while and other options.
---                                There's a default function which can be overriden with this parameter.
---     * on_full_scan_start    -- Function to call before starting a full scan iteration.
---     * on_full_scan_complete -- Function to call after completing a full scan iteration.
---     * on_full_scan_success  -- Function to call after successfully completing a full scan iteration.
---     * on_full_scan_error    -- Function to call after terminating a full scan due to an error.
---     * args                  -- Passed to is_tuple_expired and
---                                process_expired_tuple() as additional context.
---     * full_scan_time        -- Time required for a full index scan (in seconds).
---     * iteration_delay       -- Max sleep time between iterations (in seconds).
---     * full_scan_delay       -- Sleep time between full scans (in seconds).
---     * force                 -- Run task even on replica.
---  }
+--- Run a scheduled task to check and process (expire) tuples in a given space.
+--
+-- @string name
+--     Task name.
+-- @string space_id
+--     Space to look in for expired tuples.
+-- @func is_tuple_expired
+--     Function, must accept tuple and return true/false (is tuple expired or
+--     not), receives `args`, `tuple` as arguments.
+-- @table[opt] options
+--     Table with named options.
+-- @table[opt] options.args
```

Comment:
Perhaps it is worth putting, at the beginning, the parameters that have a stronger effect on the customization of the task. For example, the index is often searched for and asked in the public Tarantool chat

Reply:
Options sorted alphabetically order by name. It's allow to find parameters that have strong effect on the customization personally for you ;)